src/nano_contracts/serializer.ts

```diff
@@ -22,6 +22,7 @@ class Serializer {
    * @memberof Serializer
    * @inner
    */
+  // eslint-disable-next-line class-methods-use-this -- XXX: This method should be made static
```

Comment:
Maybe disable this on the whole class

Reply:
Good point. Done on https://github.com/HathorNetwork/hathor-wallet-lib/pull/679/commits/2c120faba11bf808b123074a66bbf743748d3d75